src/NinjaTools.FluentMockServer/Xunit/ContextRegistry.cs

```diff
@@ -0,0 +1,43 @@
+using System;
+using System.Collections.Concurrent;
+using System.Linq;
+using System.Reflection;
+using JetBrains.Annotations;
+using Xunit;
+
+namespace NinjaTools.FluentMockServer.Xunit
+{
+    public class ContextRegistry
+    {
+        private static readonly Lazy<ContextRegistry> _factory = new Lazy<ContextRegistry>(() => new ContextRegistry());
+        public static ContextRegistry Instance => _factory.Value;
+        private ContextRegistry(){}
+
+        public ConcurrentDictionary<MethodInfo, string> Collections { get; } = new ConcurrentDictionary<MethodInfo, string>();
+        public ConcurrentBag<MethodInfo> Isolated { get; } = new ConcurrentBag<MethodInfo>();
+
+        internal bool TryGetIdentifier(MethodInfo mi, out string id)
+        {
+            if (!Isolated.Contains(mi))
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/manual/git_36/NinjaTools.FluentMockServer/pullRequest?prid=4885942)